src/views/iframe.vue

```diff
@@ -100,6 +127,7 @@ export default {
       for (let option of this.options) {
         externalScript.setAttribute(option, "")
       }
+      externalScript.setAttribute(this.selectedTheme, "")
```

Comment:
```suggestion
      externalScript.setAttribute("theme", this.selectedTheme)
```

me parait beaucoup compréhensible.

J'avais cru comprendre que cette ligne était un _reset_ de l'attribut alors que c'est un _set_.